packages/ariakit/src/combobox/__examples__/combobox-highlighted-text/style.css

```diff
@@ -0,0 +1,5 @@
+@import "../combobox/style.css";
+
+[data-user-value] {
+  color: palevioletred;
```

Comment:
Maybe I should use a more proper color from tailwind theme 🤔 

I'll check it later